DigitalLearningSolutions.Data/Helpers/LearningHubSsoSecurityHelper.cs

```diff
@@ -0,0 +1,62 @@
+﻿namespace DigitalLearningSolutions.Data.Helpers
+{
+    using System;
+    using System.Security.Cryptography;
+    using System.Text;
+    using DigitalLearningSolutions.Data.Services;
+
+    public class LearningHubSsoSecurityHelper
+    {
+        private static readonly DateTime UnixEpoch = new DateTime(1970, 1, 1, 0, 0, 0, DateTimeKind.Utc);

```

Comment:
It might be worth noting on the ticket or clarifying with LH that times should be based off this epoc.
The documentation is not explicit on this but we can imply it based on their examples. 
I think it's worth ticking this box and getting clarification.